test/specs/run-spec.js

```diff
@@ -16,6 +16,9 @@ function runSpecs(title, dir, showCompletionTable, options) {
           spec.options = Object.assign({}, options, (spec.options || {}));
           const example = (spec.example ? ' example ' + spec.example : '');
           const passFail = (spec.shouldFail ? 'fail' : 'pass');
+          if (spec.options.sanitizerRemoveHtml) {
+            spec.options.sanitizer = () => '';
+          }
```

Comment:
Could we change this to just use sanitizer instead of creating a new option

somthing like:

```js
if (spec.options.sanitizer) {
  // eslint-disable-next-line no-eval
  spec.options.sanitizer = eval(spec.options.sanitizer);
}
```

then the frontmatter would just look like:

```markdown
---
sanitize: true
sanitizer: () => ''
---
```

Reply:
Yeah sure, I was thinking about that also, but my security-oriented mind screams every time I have to write down `eval`... 🙈

But we are talking here about only tests, so it's probably okay...

Reply:
Modified the code according your suggestion.